src/call/pairwise.rs

```diff
@@ -179,11 +187,34 @@ pub fn call<A, B, P, M, R, W, X, F>(
                 for (i, pileup) in pileups.iter().enumerate() {
                     if let &Some(ref pileup) = pileup {
                         for obs in pileup.case_observations() {
-                            outobs.encode((chrom, record.pos(), i, "case", obs))?;
+                            outobs.serialize(
```

Comment:
So, it now cannot deal with nested structs in tuples anymore? This sounds like a serious downside to me. Is it not possible to stick with the old csv version, while the rust-bio crate is using the new one?

Reply:
According to BurntSushi/rust-csv#75 nested stuff should work in theory, just not with automatic header writing? Or is that an old version before the switch to serde?

Reply:
As for writing nested structs: From what I gather from the [new `serialize` docs here](https://docs.rs/csv/1.0.0-beta.5/csv/struct.Writer.html#structs), `struct` entries have to be named (which ours are) and all scalars (which our `enum` isn't). So unless I am missing something, we cannot use automatic header generation with our setup, unless we use the redefined `struct` I suggested.

But as in the link you provided, we could just go back to using `has_header(false)` in the `WriterBuilder` and explicitly writing the header line. Then, it can handle the nesting for writing. I have just tested this and could quickly clean it up and check it in, right away. Would you prefer that?

Reply:
Yes, this is a good solution. Just write the header manually, and then we are able to use the previous way of writing the observation.